core/const.py

```diff
@@ -784,7 +784,7 @@ class ITLRegion(StrEnum):
 # form version and reporting period for different rounds
 GET_FORM_VERSION_AND_REPORTING_PERIOD = {
     3: ("Town Deals and Future High Streets Fund Reporting Template (v3.0)", "1 October 2022 to 31 March 2023"),
-    4: ("Town Deals and Future High Streets Fund Reporting Template (v4.0)", "1 April 2023 to 30 September 2023"),
+    4: ("Town Deals and Future High Streets Fund Reporting Template (v4.2)", "1 April 2023 to 30 September 2023"),
```

Comment:
This also needs to be reflected in PRETRANSFORMATION_FAILURE_MESSAGE_BANK. The messages are currently specific to R4, and operate under the presumption of a static reporting period and form version.

Reply:
Ah - thanks will change this